### ISSUE TITLE
Use prefers-color-scheme for system dark mode detection

### ISSUE THREAD

**Issue by GuessWhoSamFoo** [vmware-tanzu/octant#1926]
**Describe the problem/challenge you have**
Systems can have settings to select dark mode. Octant can try to detect the theme in `theme.service.ts` and automatically set defaults this way.

**Describe the solution you'd like**
See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by mklanjsek** [comment]
Already done, waiting for #1538 to get merged 😃

**Comment by mklanjsek** [comment]
This is included now after #1538 is merged. We are detecting the system theme preference and use it as Octant default theme.
